packages/tocco-ui/src/Panel/StyledPanelBody.js

```diff
@@ -1,8 +1,22 @@
 import styled from 'styled-components'
+import {theme} from 'styled-system'
 
 const StyledPanelBody = styled.div`
-  height: ${props => props.isOpen ? props.heightIfOpen : '0'};
-  overflow-y: auto;  /* if calculation does not work properly content is still accessible */
-  transition: height 300ms ease-in-out;
+  && {
+    height: ${props => props.isOpen ? props.heightIfOpen : '0'};
+    overflow-y: auto;  /* if calculation does not work properly content is still accessible */
```

Comment:
the scrollbar is shown during the animation which doesn't look very nice in my opinion. should we make sure that the scrollbar isn't shown during the animation? what do you think?

Reply:
I agree. see c5ae6d074a895b0110a716a30acb4b22ca7a1eb2